frontend/packages/kubevirt-plugin/integration-tests-cypress/views/wizard.ts

```diff
@@ -57,7 +57,13 @@ const fillBootSource = (
       break;
     }
     default: {
-      break;
+      // break;
+      cy.get(wizardView.sourceURL).type(provisionSource.getSource());
```

Comment:
The test failed because here. Suggest to change the code a bit:
```  switch (provisionSource.getDescription()) {
    case 'URL': {
      cy.get(wizardView.sourceURL).type(provisionSource.getSource());
      if (pvcSize) {
        cy.get(wizardView.pvcSize)
          .clear()
          .type(pvcSize);
      }
      break;
    }
    case 'Clone': {
      cy.get('body').then(($body) => {
        if ($body.find(wizardView.clonePVCNSDropdown).length) {
          cy.get(wizardView.clonePVCNSDropdown).click();
          cy.get(wizardView.projectNS(pvcNS)).click();
          cy.contains(wizardView.clonePVCDropDown, 'Select Persistent Volume Claim').click({
            force: true,
          });
          cy.get(wizardView.pvcName(pvcName)).click();
        } else {
          cy.get(wizardView.pvcNSDropdown).click();
          cy.get(wizardView.projectNS(pvcNS)).click();
          cy.get(wizardView.pvcDropdown).click();
          cy.get(wizardView.pvcName(pvcName)).click();
        }
      });
      break;
    }
    case 'Registry': {
      cy.get(wizardView.sourceRegistry)
        .clear()
```

Reply:
done